bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/Car.java

```diff
@@ -105,6 +117,13 @@ public void setBatteryStatus(JsonObject responseJson) {
                 if (attributes.get("chargingRemainingTime") != null) {
                     chargingRemainingTime = attributes.get("chargingRemainingTime").getAsInt();
                 }
+                if (attributes.get("timestamp") != null) {
+                    try {
+                        batteryStatusUpdated = ZonedDateTime.parse(attributes.get("timestamp").getAsString());
+                    } catch (DateTimeParseException e) {
+                        logger.warn("Error updating battery status updated timestamp. {}", e.getMessage());
```

Comment:
Log level should be reduced to **debug**. Otherwise users will see this for every poll cycle and all refresh commands in case the service would start to return incompatible values.

Reply:
Additionally, should `batteryStatusUpdated` be set to `null` in this case?

Reply:
Done

Reply:
I set these to null now.  The channel will not get updated if the values are null.  So the last update in the Item value will be the last successful update.